index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Mobile First Project</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"/>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+
+    <header>
+            <nav>
+                <ul class="header">
+                    <li class="time">9:41</li>
+                    <li class="cellular"><img src="./image/cellular.png" alt=""></li>
+                    <li class="wifi"><img src="./image/Wifi.png" alt=""></li>
+                    <li class="battery"><img src="./image/battery.png" alt=""></li>
```

Comment:
> |Figma| Yours |
> |---|---|
> | <img width="311" alt="im4" src="https://user-images.githubusercontent.com/77843050/204768909-01914d1e-0a12-447f-87d3-22f5d021922f.png"> | <img width="376" alt="paul" src="https://user-images.githubusercontent.com/77843050/204769095-6077c851-d2c3-46cf-8ec7-51f031932576.png"> |

- [ ] I salute your attention to detail but am sorry to inform you that this part from the line `17` to line `20` should not be included in the design, please remove it and only keep the nav with your logo name and hamburger
- [ ] Right after the `navbar`, the space below in your image is not well adjusted, you are required to use the same padding, margin, and everything else referring to this design.
- [ ] the `background` of your page is a bit more black than the original design, yes, I think we can make the difference between them, please correct that one by using the same color as per the Figma design.
- [ ] Let's take a closer look at these 2 images, well, the design is fine but I still think that it can be modified and improved, the `texts` font is not the same as the original design font, please make sure to use the same properties as per the Figma design.
- [ ] What else, well now are at the bottom of the page, the white space highlighted in red should not be there please make some touches to cover the whole part with your page as per the Figma design.
- [ ] The last thing is the white spaces on the left and right of your page, please adjust by removing them and sticking to the Figma design.